pkg/domain/entities/container_ps.go

```diff
@@ -15,6 +15,8 @@ type ListContainer struct {
 	Command []string
 	// Container creation time
 	Created int64
+	// Human readable container creation time.
+	CreatedAt string
```

Comment:
I wanted to point out that the `Created` and `CreatedAt` fields are switched between `ps` and `images`.

In `images`, `Created` is the human-friendly string while `CreatedAt` is the machine-readable form. In `ps`, they are the other way round.

I don't really mind it. Just wanted to be sure that it's a deliberate decision (for Docker compatibility?).

Reply:
Do  you know if they are switched in Docker as well?

Reply:
Docker does not offer a JSON-formatted output.